lib/OctokitAction.ts

```diff
@@ -83,8 +83,13 @@ export abstract class OctokitAction extends Action {
   }
 
   protected async reassignIssue(issue: { number: number }, login: string): Promise<void> {
-    await this.removeAssignees(issue);
-    await this.addAssignee(issue, login);
+    if (login.indexOf("[bot]") === -1) {    // Avoid dependabot[bot]
```

Comment:
```suggestion
    if (! login.includes("[bot]")) {    // Avoid dependabot[bot]
```

I prefer [`String.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes), WDYT?